### ISSUE TITLE
BraninNormalized function

### ISSUE THREAD

**Issue by langongjin** [resibots/limbo#287]
hi, I do not understand the BraninNormalized function in the code.

```cpp
double operator()(const vec_t& x) const
    {
        double x1 = x(0) * 15 - 5;
        double x2 = x(1) * 15;
        double term1 = sqr(x2 - (5.1 * sqr(x1) / (4. * sqr(M_PI))) + 5. * x1 / M_PI - 6);
        double term2 = (10. - 10. / (8. * M_PI)) * std::cos(x1);
        return (term1 + term2 - 44.81) / 51.95;
    }
```

It is different from the original Branin function https://www.sfu.ca/~ssurjano/branin.html

Why it is (term1 + term2 - 44.81)/51.95? what is the meaning 44.81, 51.95?

I checked the BraninNormalized function on BayesOpt at:
https://rmcantin.bitbucket.io/html/testfunctions_8hpp_source.html
The BraninNormalized function on BayesOpt is the same as the original Branin function.

Thanks!

**Comment by costashatz** [comment]
In the link that you provide (https://www.sfu.ca/~ssurjano/branin.html), if you check further down you will find a section called **Modifications and Alternate Forms** for a branin function inside [0,1]^2: *This rescaled form of the function has a mean of zero and a variance of one*

**Comment by langongjin** [comment]
yes, thanks! But still a problem, when I change the function to (- term1 - term2 - 10). You know this is the maximum the Branin function. But the results are always not right. for example, 
```cpp
branin
Result: 0.74869176446679853 0.99997459736828731 -> 212.82484989649009322
Smallest difference: 212.42696253873742762
```
I run this for many time, all of them are wrong.
